tests/product/base_product_case.py

```diff
@@ -267,8 +267,19 @@ def get_file_content(self, host, filepath):
 
     def assert_config_perms(self, host, filepath):
         self.assert_file_perm_owner(
+            host, filepath, '-rw-r--r--', 'presto', 'presto')
+
+    def assert_connnector_perms(self, host, filepath):
+        self.assert_file_perm_owner(
             host, filepath, '-rw-------', 'presto', 'presto')
 
+    def assert_directory_perm_owner(self, host, filepath, permissions, owner, group):
+        ls = self.cluster.exec_cmd_on_host(host, "ls -l -d %s" % (filepath,))
```

Comment:
Everything but the ls command is the same in this and assert_file_perm_owner. Please consolidate.